apps/turbo-remote-cache/.cloud-foundry/manifest-prod.yml

```diff
@@ -0,0 +1,14 @@
+applications:
+  - name: turbo-remote-cache
+    instances: 2
+    memory: 256MB
```

Comment:
Are this the recommended settings for the docker image? Because Cloud Foundry charges per memory/RAM increasing it unnecessary might blow up the costs :)

Reply:
Unfortunately, there are no recommend settings provided.
I played around with different memory settings, 128 MB also seemed to work fine.
Cloud Foundry then showed a max RAM usage of around 40-50%. 
So I would like to stay with 256 MB as we will have also way more workload after the merge.
Though, we can check the Cloud Foundry metrics after a week of real usage and still decide to downscale.

Anyway, the budget for the current setting was approved by @mj-hof.